tests/test_exif.py

```diff
@@ -10,6 +10,10 @@ def test_exif_reads_valid_file():
     exif = Exif("input/exif.jpg")
     assert exif.data()['CreateDate'] == '2017:01:01 01:01:01'
 
+def test_exif_reads_files_with_illegal_characters():
+    exif = Exif("input/!#$%&'\"*+-.^_`|~:.jpg")
+    assert exif.data()['CreateDate'] == '2017:01:01 01:01:01'
```

Comment:
How come a non-existent file provides a valid `CreateDate`?

Reply:
Sorry, it's like I forgot to include this file in the commit. Fixed.